skiplang/prelude/src/skstore/Handle.sk

```diff
@@ -258,6 +258,60 @@ fun sumReducer(): EReducer<IntFile, IntFile> {
   };
 }
 
+fun minReducer(): EReducer<IntFile, IntFile> {
+  EReducer{
+    type => IntFile::type,
+    canReset => true,
+    init => iter ~> {
+      min: ?Int = None();
+      for (x in iter) {
+        if (min.isNone() || x.value < min.fromSome()) !min = Some(x.value);
+      };
+      min match {
+      | Some(x) -> Array[IntFile(x)]
+      | None() -> Array[]
+      }
+    },
+    update => (state, old, new) ~> {
+      min = state[0].value;
+      for (x in new) {
+        if (x.value < min) !min = x.value;
+      };
+      for (x in old) {
+        if (x.value < min) return None()
```

Comment:
Shouldn't this be `<=`?

Reply:
Yeah, nice catch, thanks! Made the change. (and for max too)